js/apps/events-page/dataServices/eventsService.js

```diff
@@ -0,0 +1,54 @@
+((app) => {
+	const eventsService = (CONSTANTS, $http, $q) => {
+		const isEventOnDate = (eventItem, eventDate) => {
+			const eventItemDate = (new Date(eventItem.EventStart)).toLocaleDateString();
```

Comment:
I think the name of these variables could be slightly more clear. eventItemDate is really ```itemStartDateLocaleString```? and event date is really ```eventStartDateAsString``` or ```eventStartLocaleString```

Reply:
Done!